mathesar/exceptions/exceptions.py

```diff
@@ -0,0 +1,49 @@
+from collections import namedtuple
+from typing import List
+
+from django.utils.encoding import force_str
+from rest_framework import status
+from rest_framework.exceptions import APIException
+
+from mathesar.exceptions.error_codes import ErrorCodes
+
+ExceptionBody = namedtuple('ExceptionBody',
+                           [
+                               'code',
```

Comment:
Per the [spec](https://wiki.mathesar.org/en/engineering/architecture/api-standards), this should be `error_code`.

Reply:
Made an intentional change as `drf` seems to be using `code` rather than `error_code`. Moreover `code` seems apt as it is already inside an `error object`

Reply:
Okay, that's fine with me, but please update the spec too.